docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/ootb-data/user-and-session-identification/index.md

```diff
@@ -0,0 +1,177 @@
+:::
+
+### Server (Collector) generated user identifier
+
+The Snowplow Collector generates a user identifier that is stored in cookies for the Collector domain. This is the `network_userid` property in the atomic events table and it has the format of a UUID.
+
+The identifier is available both in Web and mobile apps. However, in Android apps, it is stored in memory so it is reset after the app restarts.
+
+In most scnearios, this identifier may have a longer lifetime than the tracker generated identifier. However, browsers can restrict it's lifetime for different reasons, such as when the Snowplow Collector is on a third-party domain from the website (not recommended), or due to the ITP restrictions in Safari (Snowplow provides a solution to mitigate this problem – [the ID service](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/advanced-usage/using-an-id-service/index.md)).
+
+:::info
+`network_userid` is captured via a cookie set by the Snowplow Collector. It can be overriden by setting `tnuid` on a Tracker request payload but is typically expected to be populated by the Collector cookies.
+:::
+
+### Business user identifier
+
+This is an external identifier given in the tracker by the app. Most commonly it is the username or e-mail address of the logged in user (e.g., `jon.doe@email.com`).
+
+All our trackers have an API to set this identifier. You can find it in [the JavaScript tracker docs here](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracker-setup/additional-options/index.md#setuserid) and [the mobile trackers docs here](/docs/collecting-data/collecting-from-own-applications/mobile-trackers/client-side-properties/index.md#set-the-subjectconfiguration).
+
+This identifier can be very useful to stitch the generated tracker identifiers together in order to identify the same user across multiple browsers or devices. See below for more information on user stitching.
+
+### IP address
+
+Although it's not a very reliable identifier, the events also contain the IP address of the user which can be used to identify them to some extent.
+The value is stored in the `user_ipaddress` property in the atomic events table.
+
+## Session identifiers
+
+In Snowplow events, sessions are tracked on the client-side – by the tracker.
+The session information is then attached to the events when they are tracked.
+
+The session information consists of two main pieces of information (when using the client session entity discussed below, there is also extra information tracked with sessions):
+
+1. **Session identifier**. UUID identifier generated by the tracker at the start of the session.
+2. **Session index**. Index of the session for the current user (per the tracker generated user identifier).
+
+Sessions have a configurable inactivity timeout.
```

Comment:
Is there a default / suggested default to mention here?

Reply:
Good point, have mentioned the 30 mins default.